x-pack/test/security_solution_cypress/cypress/e2e/explore/cases/creation.cy.ts

```diff
@@ -48,16 +48,21 @@ import {
   fillCasesMandatoryfields,
   filterStatusOpen,
 } from '../../../tasks/create_new_case';
-import { login } from '../../../tasks/login';
 import { visit, visitWithTimeRange } from '../../../tasks/navigation';
 
 import { CASES_URL, OVERVIEW_URL } from '../../../urls/navigation';
 import { ELASTICSEARCH_USERNAME, IS_SERVERLESS } from '../../../env_var_names_constants';
 import { deleteCases } from '../../../tasks/api_calls/cases';
+import { login } from '../../../tasks/login';
 
-// https://github.com/elastic/kibana/issues/179231
 const isServerless = Cypress.env(IS_SERVERLESS);
-const username = isServerless ? 'platform_engineer' : Cypress.env(ELASTICSEARCH_USERNAME);
+const getUsername = () => {
```

Comment:
This looks like something that is going to be reused in the future. Should we convert this into a Cypress command, like `cy.getUsername()`?

Reply:
I put it there since I saw that only this test was using that functionality.